InfectedRose.Database/InfectedRose.Database.csproj

```diff
@@ -15,6 +14,7 @@
 
     <ItemGroup>
       <PackageReference Include="Microsoft.Data.Sqlite" Version="3.1.2" />
+      <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
```

Comment:
This change is required as part of .NET Standard 2.1 since the `System.ComponentModel.Annotations.Schema` namespace is required, but `System.ComponentModel.Annotations` is not included in .NET Standard. It is part of .NET Core and .NET though.